{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-1002105",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://www.securityfocus.com/bid/106068",
          "name": "106068",
          "refsource": "BID",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3537",
          "name": "RHSA-2018:3537",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3549",
          "name": "RHSA-2018:3549",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3551",
          "name": "RHSA-2018:3551",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3598",
          "name": "RHSA-2018:3598",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3624",
          "name": "RHSA-2018:3624",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3742",
          "name": "RHSA-2018:3742",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3752",
          "name": "RHSA-2018:3752",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://access.redhat.com/errata/RHSA-2018:3754",
          "name": "RHSA-2018:3754",
          "refsource": "REDHAT",
          "tags": [

          ]
        },
        {
          "url": "https://github.com/kubernetes/kubernetes/issues/71411",
          "name": "https://github.com/kubernetes/kubernetes/issues/71411",
          "refsource": "CONFIRM",
          "tags": [

          ]
        },
        {
          "url": "https://groups.google.com/forum/#!topic/kubernetes-announce/GVllWCg6L88",
          "name": "https://groups.google.com/forum/#!topic/kubernetes-announce/GVllWCg6L88",
          "refsource": "CONFIRM",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In all Kubernetes versions prior to v1.10.11, v1.11.5, and v1.12.3, incorrect handling of error responses to proxied upgrade requests in the kube-apiserver allowed specially crafted requests to establish a connection through the Kubernetes API server to backend servers, then send arbitrary requests over the same connection directly to the backend, authenticated with the Kubernetes API server's TLS credentials used to establish the backend connection."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-12-05T21:29Z",
  "lastModifiedDate": "2018-12-06T11:29Z"
}